using-timescaledb/hypertables.md

```diff
@@ -69,6 +69,17 @@ DROP TABLE conditions;
 
 ---
 
+### Distributing a Hypertable across multiple nodes [](distributing)
+---
+
+To create a *distributed hypertable* that scales out across many
+nodes, execute the TimescaleDB command `create_distributed_hypertable`
+instead of the regular `create_hypertable` command ([API
```

Comment:
Suggestion: "Hypertables can be distributed over several nodes using the `create_distributed_hypertable` command instead of the `create_hypertable` command." This will implicitly introduce the notion of a distributed hypertable, so I do not think you need a definition for it.